tools/TimeFileMaker.py

```diff
@@ -108,6 +122,7 @@ def format_percentage(num, signed=True):
     return sign + '%d.%02d%%' % (whole_part, frac_part)
 
 def make_diff_table_string(left_times_dict, right_times_dict,
+                           sort_by='auto',
```

Comment:
Why make this optional?

Reply:
You mean, why specify the default in two places, rather than one?  I guess there's no particularly good reason to do that, here.

Reply:
Actually, hmm, one reason is that there's a difference between the wrapper script and the internal implementation, and it seems cleaner to specify the defaults separately.  Like, there is a reasonable default for `sort_by`, and it is `'auto'`, and even if all of the wrapper scripts currently pass that explicitly, it seems cleaner to consider optional/defaulted values separately for the wrappers and for the core functionality.  (For example, I could imagine switching the wrappers to use argparse, and passing in a dictionary of values via `**`, and I could also imagine writing a wrapper that doesn't take a `--sort-by` option, and both of these changes seem nicer if I can rely on the underlying method to fill in the default.)